Makefile

```diff
@@ -439,6 +441,13 @@ codecov:
 	bash <(curl -s https://codecov.io/bash)
 .PHONY: codecov
 
+.clang-format: .FORCE
+	scripts/fetch-clang-format.sh
+
+indent:
```

Comment:
Should it depend on `.clang-format`?
If not, maybe it should be a target instead of file-rule, like `fetch-clang-format`?
No hard feelings though.

Reply:
changed